src/main/java/teammates/ui/controller/AdminEmailImageUploadAction.java

```diff
@@ -43,15 +43,15 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
       
         data.isFileUploaded = true;
-        data.fileSrcUrl = Config.APP_URL + 
-                          Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
+        data.fileSrcUrl = Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
```

Comment:
I'd rather not do it now. Let's do it along with hundreds other URLs out there that are also constructed manually (well maybe not hundreds, but there is a lot).